Update node details and legend in Topology View.

kialam opened the pull request:
##### SUMMARY
<!--- Describe the change, including rationale and design decisions -->

<!---
If you are fixing an existing issue, please include "related #nnn" in your
commit message and your description; but you should still explain what
the change does.
-->
Update node details and legend in Topology View.
![Screen Shot 2022-08-21 at 8 16 29 PM](https://user-images.githubusercontent.com/2293210/185831960-e917de26-f4cd-4567-8c5e-68ecc57b9a65.png)

##### ISSUE TYPE
<!--- Pick one below and delete the rest: -->
 - New or Enhanced Feature

##### COMPONENT NAME
<!--- Name of the module/plugin/module/task -->
 - UI

kialam committed:
Add new node details; update legend.

AlexSCorey commented on awx/ui/src/screens/TopologyView/Legend.js:
```diff
@@ -70,9 +71,7 @@ function Legend() {
-            <Button variant="primary" isSmall>
-              {t`C`}
-            </Button>
+            <Button isSmall>{t`C`}</Button>
```
Just thinking about how this is going to be translated.  I think we should add an aria-label here to help with accessibility.

AlexSCorey commented on awx/ui/src/screens/TopologyView/Tooltip.js:
```diff
@@ -71,36 +184,130 @@ function Tooltip({
+                            value={instanceDetail.capacity_adjustment}
+                            onChange={handleChangeValue}
+                            isDisabled={!instanceDetail.enabled}
+                            // isDisabled={!me?.is_superuser || !instance.enabled}
```
Remove?

AlexSCorey reviewed (approved):
This looks good.  I asked a question offline about a translation question.  I don't really think its 
 a blocker but more of something we need to consider.  Just remove the commented code and we are good to go.

kialam committed:
Represent `enabled` field in Topology View:

- use dotted circles to represent `enabled: false`
- use solid circle stroke to represent `enabled: true`
- excise places where `Unavailable` node state is used in the UI.

kialam committed:
Add correct permissions for memory capacity slider.
